include/openPMD/IO/ADIOS/macros.hpp

```diff
@@ -19,6 +19,11 @@
 #define openPMD_HAS_ADIOS_2_9                                                  \
     (ADIOS2_VERSION_MAJOR * 100 + ADIOS2_VERSION_MINOR >= 209)
 
+#if !defined(ADIOS2_HAVE_BP5) && openPMD_HAS_ADIOS_2_9
+// ADIOS2 v2.10 no longer defines this
+#define ADIOS2_HAVE_BP5
```

Comment:
Let's call this `openPMD_HAS_ADIOS_BP5` and add all logic here (and rename other uses).